crates/beerus-core/src/lightclient/ethereum/helios_lightclient.rs

```diff
@@ -207,22 +212,26 @@ impl HeliosLightClient {
     /// Create a new HeliosLightClient.
     pub async fn new(config: Config) -> eyre::Result<Self> {
         // Build the Helios wrapped light client.
-        #[cfg(feature = "std")]
-        let helios_light_client: Client<FileDB> = ClientBuilder::new()
+        let mut builder = ClientBuilder::new()
             .network(config.ethereum_network()?)
             .consensus_rpc(config.ethereum_consensus_rpc.as_str())
             .execution_rpc(config.ethereum_execution_rpc.as_str())
-            .load_external_fallback()
-            .data_dir(config.data_dir)
-            .build()?;
+            .load_external_fallback();
+
+        if cfg!(feature = "std") {
+            builder = builder.data_dir(config.data_dir.clone());
+            builder = HeliosLightClient::load_checkpoint(
+                builder,
+                config.helios_checkpoint,
+                config.data_dir.clone(),
```

Comment:
I have to clone here because helios functions take ownership. Something better I can do here, of the `clone()` is acceptable?